src/Blog/View/post.twig

```diff
@@ -1,21 +1,55 @@
+
+Duis ultrices, velit interdum interdum sagittis, orci enim pulvinar sem, at congue enim velit eget lectus.  <br> <br> Nullam quis venenatis nisi. Phasellus scelerisque nisl nec odio sollicitudin bibendum. Duis ante sem, suscipit quis neque ac, viverra lobortis massa. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Aliquam lacinia blandit orci, et euismod turpis lobortis nec. Nulla molestie eros vitae urna luctus, vel fringilla augue efficitur. Mauris ac nisi sagittis, posuere nisi quis, varius lectus. Vestibulum sagittis maximus dui ac accumsan. Etiam eros odio, dignissim eget ipsum id, ultricies commodo massa. Morbi mattis fringilla auctor. Nullam suscipit ante purus, vitae blandit neque vehicula ut. Pellentesque viverra nulla a leo laoreet rutrum. Proin venenatis, mauris et ultrices pulvinar, nibh dolor scelerisque tortor, nec aliquet elit enim vel nulla.
+
+Nunc venenatis lacus ac ex mattis efficitur. Sed nec quam ut nisl auctor commodo varius vel purus. Pellentesque facilisis tristique leo, eget egestas mi lobortis suscipit. Fusce lacinia magna interdum, fringilla orci sed, auctor augue. Quisque ultricies nibh vel massa feugiat, et dictum massa luctus. Curabitur id iaculis ligula, in mattis augue. Aliquam erat volutpat. Cras eu urna semper, varius ante ullamcorper, pulvinar leo. ',
     'url':'/img/slider1.svg',
     'date':'10-02-2020',
     'author_forename': 'John',
-    'author_surname': 'Nowak'
+    'author_surname': 'Nowak',
+	'author_position': 'Frontend Developer',
 }%}
 {% include 'Common/View/header.twig' %}
 {% block body %}
 	<body class="top-wave">
 	{% endblock %}
 	<section class="container">
-		<h1>{{ post.title }}</h1>
-		<p>{{ post.date }}</p>
-		<p>{{ post.author_forename }}
-			{{ post.author_surname }}</p>
-		<p>{{ post.content|raw }}</p>
+		<img src="/img/BlogPostHeader.jpg" alt="Blog Post Header" class="post-page__img">
+
+		<article class="u-margin--medium"> 
+			<div class=".post-page__header clearfix"> 
+			<h1 class="post-description__title">{{ post.title }}</h1>
+			<span class="post-description__date">{{ post.date }}</span> 
+			</div>
+			<p>{{ post.content|raw }}</p>
+		</article>
+
+		<div class="post-end clearfix">
+			<a class="read-previous-button post-page-end__button--previous clearfix">
+				<i class="arrow left"></i>
+				<span>poprzedni artykuł </span>
+			 </a>
+
+			<div class="post-page__author-info clearfix"> 
+				<img src="/img/bojack_headshot.jpg" alt="Author's Headshot" class="author-box__img">
+				<div class="author-box__description"> 
+					<p class="author-title">AUTHOR:</p>
+					<p class="author-info--name"> {{ post.author_forename }} {{ post.author_surname }} </p>
+					<p class="author-info--position"> {{ post.author_position }}</p>
+				</div>
+			</div>
+
+			<a class="read-more-button post-page-end__button--next clearfix"> 
+				<span>następny artykuł </span>
+				<i class="arrow right"></i>
```

Comment:
I see that you use external icon. That good for you :)
I didn't tell you what icon you can use, and where they are, so you manage it great. 
If you want to come colose to mockup desing appearance, you might use icon, which I add to trello. icon-back.svg. Firstly, you need to add this icon to your repositorium in `public/img` direction. There is only one icon in back direction. I deal with it be transofrme it and rotate it in 180deg. 
```
{ transform: rotate(180deg)}
```

Reply:
And I see that you did the same transformation with your icon ;) that is cool 👍

Reply:
I just used the simplest solution which I found here: https://www.w3schools.com/howto/howto_css_arrows.asp :)